scripts/github_issue_retriever.py

```diff
@@ -66,8 +66,9 @@ def retrieve_issues(self) -> None:
 
         try:
             last_modified = db.last_modified(self.github.db_path)
-        except Exception:
-            pass
+        except Exception as e:
+            if str(e) == "Last-Modified is not available":
```

Comment:
The correct behavior was the previous one. We are in the script which is generating the DB. If a DB with the current schema doesn't exist, we simply want to start from scratch (instead, by `return`ing, you are terminating the generation of the DB).

Reply:
Ok

Reply:
pass for any exception ?

Reply:
Good point. No, only if the exception is "Last-Modified is not available".